index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
```

Comment:
Let me explain this a bit to try to clarify your confusion. 
In this project, you are supposed to make a code review of your past project, so you have to copy your last project (the one you did in the trials) and make a code review by yourself, once you do that, you can ask for a review, so I or any other code reviewer can see your pull request.
So please, in order to proceed with this project, create a new review of this project and then send the link to the pull request again.